backend/pah_fm/settings.py

```diff
@@ -21,6 +21,7 @@
 
 CSRF_COOKIE_SECURE = True
 CSRF_COOKIE_DOMAIN = BASE_URL
+CSRF_TRUSTED_ORIGINS = ["http://localhost:8080"]
```

Comment:
I'm wondering if this should also have 127.0.0.1 thing in it as well. Or maybe we should just ditch IP address and use localhost everywhere.